wts/utils.py

```diff
@@ -0,0 +1,41 @@
+import flask
+import json
+import os
+
+
+def get_config_var(variable, default=None, secret_config={}):
+    """
+    get a secret from env var or mounted secret dir,
+    raise exception if it doesn't exist
+    """
+    path = os.environ.get("SECRET_CONFIG")
+    if not secret_config and path:
+        with open(path, "r") as f:
+            secret_config.update(json.load(f))
+    value = secret_config.get(variable.lower(), os.environ.get(variable)) or default
+    if value is None:
+        raise Exception(
+            "{} configuration is missing, abort initialization".format(variable)
+        )
+    return value
+
+
+def get_oauth_client(idp=None):
+    """
+    Args:
+        idp (str, optional): IDP for the OAuthClient to return. If not
+        provided, will use the IDP provided as a request argument. By default,
```

Comment:
What do you think of refactoring this function so that the signature is `get_oauth_client(idp='default')` and make it the responsibility of the caller to pass the `idp` argument of the current flask request if it exists?

(Minor) If you don't refactor and keep the current behavior, I think you should change the comment to:

`If not provided, will use the 'idp' argument of the current flask request. If there is no 'idp' argument in the current flask request, 'idp' will be set to "default"`